src/qml/webpatchesmodel.cpp

```diff
@@ -94,9 +116,43 @@ void WebPatchesModel::componentComplete()
     connect(watcher, &QDBusPendingCallWatcher::finished, [this](QDBusPendingCallWatcher *watcher){
         QDBusPendingReply<QVariantList> reply = *watcher;
         if (!reply.isError()) {
-            const QVariantList catalog = PatchManager::unwind(reply.value()).toList();
+            QVariantList catalog = PatchManager::unwind(reply.value()).toList();
             qDebug() << Q_FUNC_INFO << catalog.count();
 
+            if (_sorted) {
+                const QLatin1String category("category");
+                const QLatin1String name("display_name");
+                const QByteArray other("other");
+                std::sort(catalog.begin(), catalog.end(),
+                    [&category, &name, &other](const QVariant &a, const QVariant &b)
+                    {
+                    const auto amap = a.toMap();
+                    const auto bmap = b.toMap();
+
+                    const auto acat = amap[category].toByteArray();
+                    const auto bcat = bmap[category].toByteArray();
+
+                    if (acat == bcat)
+                    {
+                    // If categories are equal then sort by name
+                    return compareStrings(amap[name].toString(), bmap[name].toString());
```

Comment:
so the nit about formatting is:

In this lambda the tabbing of the code inside the curly braces does not look like the rest of the file (or project - at least on cursory view). Also on line 148 it doesn't look like the rest of the function.

I do not want to start a 'code conventions' discussion here, let's go to 'ambient conventions' for the moment, what do you think?